node.gyp

```diff
@@ -675,6 +679,7 @@
         'src/util-inl.h',
         # Dependency headers
         'deps/v8/include/v8.h',
+        'test/fixtures/postject-copy/node_modules/postject/dist/postject-api.h'
```

Comment:
@mhdawson I've added a TODO for now. Do you think it can be done in a separate PR after this one lands or would you like to see this implemented before this PR lands?